src/issue-creator/cb-open-issue-creator.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpService } from '@nestjs/common';
+import { LogMessageFormat, LogType } from 'logging-format';
+import { IssueCreator } from './issue-creator';
+import { ConfigService } from '@nestjs/config';
+
+/**
+ * CbOpenIssueComponent handles CB open Logs, it extends IssueCreator to enable individual issue creation for cb open issues 
+ */
+export class CbOpenIssueCreatorComponent extends IssueCreator {
+  issueCreator: IssueCreator;
+  constructor(http: HttpService, configService: ConfigService) {
+    super(http, configService);
+    this.issueCreator = new IssueCreator(http, configService);
+  }
+
+  /**
+   * handles cp open logs by creating an Issue and sending it to the API: https://github.com/ccims/ccims-backend/tree/apiMockup
```

Comment:
das sollte cb für **c**ircuit **b**reaker sein, oder?
```suggestion
   * handles cb open logs by creating an Issue and sending it to the API: https://github.com/ccims/ccims-backend/tree/apiMockup
```

Reply:
Ja cb steht für circuit breaker